docs/docs/hedgehog-engine/wars/levels/ids.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Sonic Forces level ID map
+---
+
+# Level ID Map
+
+## Mapping
+Stage IDs are defined like this:
+```
+{world_id}{stage_type}{character_index}
+```
+!!! info "Example"
+    w5a01 -> Lost Valley
+
+### Worlds (Zones)
+
+World ID | World Name
+-------- | -----------
+w1       | Death Egg
+w2       | Chemical Plant
+w3       | City
+w4       | Mystic Jungle
+w5       | Green Hill
+w6       | Metropolis
+w7       | Eggman Empire Fortress
+
+### Stage Type
+
+Stage Type ID | Stage Type
+------------- | -----------
+a             | Act
+b             | Boss
+d             | DLC
+s             | Credits
+t             | Title
+x             | Extra
+
+### Character Index
+
+Character Index | Character
+--------------- | ----------
+01              | Modern Sonic
+02              | Avatar
+03              | Tag Team
+04              | Classic Sonic
+
+!!! note
+    Bosses don't use this value to indicate character, rather as a boss index for the current world (which only comes up in World 7).
+
+!!! note
+    The Secret and Extra stages don't use this value to indicate character.
+
+### Stages
+
+Stage Name  		 | Level ID
```

Comment:
Nitpicking, but GitHub shows inconsistent spacing in this table. Would be good if this could be fixed, but not mandatory